src/Bridges/SymfonyConsole/BaseCommand.php

```diff
@@ -24,15 +25,20 @@ abstract class BaseCommand extends Command
 	/** @var IConfiguration */
 	protected $config;
 
+	/** @var IPrinter */
+	protected $printer;
+
 
 	/**
 	 * @param  IDriver        $driver
 	 * @param  IConfiguration $config
+	 * @param  IPrinter|NULL  $printer
```

Comment:
don't we use lowercased null?

Reply:
we do now, but this is ancient code base, so it still uses uppercase `NULL`